### DISCUSSION TITLE
I have completed my GitHub user profile with full name. The school located in my city and I am not using any VPN, but I still got reject.

### DISCUSSION THREAD

**Discussion by half-drop** [community/community#54006] [GitHub Education]
### Hi everyone,

 I have completed my GitHub user profile with full name. The school located in my city and I am not using any VPN, but I still got reject.

**Comment by half-drop** [comment]
![image](https://user-images.githubusercontent.com/39797906/235270916-dde32bf2-b0c7-43de-ae2f-f2e75a669294.png)

**Reply by quanboy247** [reply]
How can I fix this

**Comment by pnpn111** [comment]
Hi! This article is my summary of the difficulties I encountered in the application. I hope it will be helpful to you!

https://www.luogu.com.cn/blog/pnpn111/github-student-packet

By the way, you seem to have given away your E-mail address.
